include/database/MysqlManager.php

```diff
@@ -554,14 +554,20 @@ public function connect(array $configOptions = null, $dieOnError = false)
             }
         }
 
-        // cn: using direct calls to prevent this from spamming the Logs
-        mysql_query("SET CHARACTER SET utf8", $this->database);
-        $names = "SET NAMES 'utf8'";
-        $collation = $this->getOption('collation');
-        if (!empty($collation)) {
-            $names .= " COLLATE '$collation'";
+	// cn: using direct calls to prevent this from spamming the Logs
+        $charset = $this->getOption('charset');
+
+        if(!empty($charset)) {
+            mysql_query("SET CHARACTER SET $charset", $this->database);
```

Comment:
Please use quote or quoted methods on variable before concatenate it into an sql query.